CONTRIBUTING.md

```diff
@@ -127,7 +127,39 @@ Run `rush rebuild` from anywhere in the repo to rebuild all projects from scratc
 
 Run `rush build -t <packagename>` to build a single project, and all local projects that it depends on. You can pass `-t` multiple times to build multiple projects. This works for `rush rebuild` as well. Keep in mind that Rush refers to packages by their full names, so packages will be named something like `@azure/<servicename>`.
 
-By default, Rush only displays things written to `STDERR`. If you want to see the full output, pass `--verbose` to any of the build commands. This is also true for script commands (see below).
+By default, Rush only displays things written to `STDERR`. If you want to see the full output, pass `--verbose` to any of the build commands.
+
+### Testing
+
+If you want to run the tests of a specific project, go to that project's folder and execute `rushx test`. All of the tests will automatically run both in NodeJS and in the browser. To target these environments individually, you can run `rushx test:node` and `rushx test:browser`.
+
+By default, these npm scripts run previously recorded tests. The recordings have been generated by using a custom recording library called [test-utils-recorder](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/test-utils/recorder/README.md). We will examine how to run recorded tests and live tests in the following sections.
+
+#### Recorded tests
+
+Most of the tests in our projects run in playback mode by default, i.e they make no network requests to the real services. For http requests made in each test case, there is a recorded response that reproduces the service behavior. The readme file in the `test` folder of each package will indicate wether the package uses recorded tests or not.
+
+#### Live tests
+
+In case you want to run the tests of the packages that use our [recorder](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/test-utils/recorder/README.md) against live resources, you will continue to run the `test` npm script, but you will need to ensure that:
+
+- The environment variable `TEST_MODE` is set to `live`.
```

Comment:
Am guessing that the reason you are calling out "packages that use our recorder" to differentiate libraries like Event Hubs and Service Bus that don't use the recorder because the TEST_MODE will not apply to them?

But rest of the instructions in this section do apply to packages that dont use the recorder. So, I would think that we don't need to call this out. Thoughts?

```suggestion
To run the tests of the packages against live resources, follow the below before running the `rushx test` command.

- Set the environment variable `TEST_MODE` to `live`.
```

Reply:
I hope you don't mind if I do this instead:

> To use the `rushx test` command to run the tests against live resources, follow the instructions below.